README.md

```diff
@@ -41,6 +41,12 @@
 
 The following commands will execute scripts to fetch and install the latest [GitHub Releases](https://github.com/fossas/fossa-cli/releases) on the corresponding operating system.
 
+### Install with Homebrew (MacOS or Linux):
+```bash
+brew tap fossas/tap
```

Comment:
[nit] We can simplify this to `brew install fossas/tap/fossa` so that users only need one line.